src/webui/www/public/scripts/client.js

```diff
@@ -310,13 +312,19 @@ window.addEvent('load', function () {
                         update_categories = true;
                     }
                     if (response['torrents']) {
+                        var newTorrentInList = false;
```

Comment:
A "new" torrent is one that is not already populated in the torrent list. It has nothing to do with the age of the torrent. Hence, it is **new** to the list.